test/helpdesk_tests/genbank_up_down_duplicate_test.py

```diff
@@ -0,0 +1,170 @@
+import os
+import time
```

Comment:
Does this module just test parsing duplicate feature ids? Can it be pared down to just be about that?